tests/test_model.py

```diff
@@ -895,11 +895,11 @@ def test_upsert_benchmarks(self):
                     provider_results=[
                         model.BenchmarkProviderResult(
                             provider='aws',
-                            value=2.0,
+                            value='2.0',
```

Comment:
I meant to ask about the change from float to string.  That's probably going to break the chart rendering and will certainly break the min / max tests in `BenchmarkResult.as_chart()` to determine the axis ranges.  Clearly the value is a float, and should be typed as such.